easybuild/easyconfigs/a/AdaptiveCpp/AdaptiveCpp-23.10.0-cpeAMD-23.09-rocm-5.4.6.eb

```
@@ -0,0 +1,102 @@
+easyblock = 'CMakeMake'
+
+name =    'AdaptiveCpp'
+version = '23.10.0'
+versionsuffix = '-rocm-5.4.6'
+
+homepage = 'https://adaptivecpp.github.io/'
+
+whatis = [
+    "Description: Implementation of SYCL and C++ standard parallelism for CPUs and GPUs from all vendors: The independent, community-driven compiler for C++-based heterogeneous programming models."
+]
+
+description = """
+AdaptiveCpp is the independent, community-driven modern platform for C++-based heterogeneous programming models targeting CPUs and GPUs from all major vendors. 
+"""
+
+toolchain = {'name': 'cpeAMD', 'version': '23.09'}
+toolchainopts = {'verbose': False, 'openmp': True}
```

```suggestion
toolchainopts = {'verbose': False, 'openmp': False}
```
During my tests, I noticed that building ACPP manually did not present the missing symbol problem. The main difference in my manual build was the absence of the `-fopenmp` flag in the `CXXFLAGS` environment variable.

I suggest disabling OpenMP here as it has not influence on the build of the OpenMP backend.